utils/build/docker/dotnet/parametric/Endpoints/ApmTestApi.cs

```diff
@@ -57,12 +58,14 @@ public static void MapApmTraceEndpoints(WebApplication app, ILogger<ApmTestApi>
     internal static readonly PropertyInfo RawTraceId = SpanContextType.GetProperty("RawTraceId", BindingFlags.Instance | BindingFlags.NonPublic)!;
     internal static readonly PropertyInfo RawSpanId = SpanContextType.GetProperty("RawSpanId", BindingFlags.Instance | BindingFlags.NonPublic)!;
     internal static readonly PropertyInfo AdditionalW3CTraceState = SpanContextType.GetProperty("AdditionalW3CTraceState", BindingFlags.Instance | BindingFlags.NonPublic)!;
+    internal static readonly FieldInfo GetW3CTraceContextPropagator = W3CTraceContextPropagatorType.GetField("Instance", BindingFlags.Static | BindingFlags.Public)!;
```

Comment:
nit: We no longer need this variable since we determined the `CreateTraceStateHeader` method is static and doesn't need an object instance to call